webapp/components/wpt-flags.js

```diff
@@ -258,6 +259,11 @@ class WPTFlagsEditor extends FlagsEditorClass(/*environmentFlags*/ false) {
         Enable Browser Specific Failures graph
       </paper-checkbox>
     </paper-item>
+    <paper-item>
```

Comment:
Is it necessary to hide it behind the flag? view=test is an optional parameter and doesn't show in the default view